pkg/logging/logging.go

```diff
@@ -65,16 +76,36 @@ func InitializeDefaultLogger() *logrus.Logger {
 	return logger
 }
 
+// GetLogLevelFromConfig returns what the log level provided via global
+// configuration. If the logging level is invalid, ok will be false.
+func GetLogLevelFromConfig() (logrus.Level, bool) {
+	lvl, ok := levelStringToLogrusLevel[strings.ToLower(options[Level])]
+	return lvl, ok
+}
+
+func setupLogLevel(logOpts map[string]string) logrus.Level {
```

Comment:
This func name is a bit confusing, as it doesn't set anything up as far as I can tell.

Maybe `getFinalLogLevel` or `getResolvedLogLevel`?

Reply:
I've renamed the function / changed some stuff around in my latest upload, let me know what you think.